tribits/python_utils/gitdist.py

```diff
@@ -722,6 +722,17 @@ def getHelpTopicsStr():
 #
 
 
+# Shrink a string to a given width by inserting an ellipsis (...) in the
+# middle.
+def shrinkString(string, width):
```

Comment:
We could unit test this function as well.  But the upper-level table shrinking tests test this.